### PR TITLE
Install OpenStack from Source part 4

### PR DESCRIPTION
**Pull request by phil-hopkins-a**
Bio:
Phil Hopkins, Principal Engineer at Rackspace. He has an Electrical Engineering Degree from the University of South Florida and has been working on Linux systems for over 15 years. He is active teaching and writing about Openstack software defined networks (SDN). He has written and currently gives an advanced class teaching OpenStack Networking.

Editing Preference:
Feel free to edit as needed.

Posting Preference:
Desired date to publish: August 11th, 2015

### CODE REVIEW

**Comment by phil-hopkins-a** [comment]
I have made the suggested changes, let me know if you see anything else. If not then this can be merged at any time.

File: _posts/2015-08-12-install-openstack-from-source4.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
+---
+layout: post
+date: 2015-08-12 23:59
```
Comment (smashwilson) [inline _posts/2015-08-12-install-openstack-from-source4.md]: Would you mind changing this (and the filename) to 2015-08-11, since it's going out today?

**Comment by phil-hopkins-a** [comment]
Made the changes.

File: _posts/2015-08-12-install-openstack-from-source4.md (hunk 1 above)
Comment (smashwilson) [inline _posts/2015-08-12-install-openstack-from-source4.md]: Er, you probably want to delete the old file, too.

**Comment by smashwilson** [comment]
> Made the changes.

I'm still seeing _both_ files in the pull request. Can you delete the 2015-08-12 version, so we don't post twice?

**Comment by phil-hopkins-a** [comment]
Its gone. My fat fingers typed the wrong thing and the file didn't go away.

**Comment by smashwilson** [comment]
Hah, no problem. I'll :shipit: as soon as Travis is green. Thanks again for contributing these :sparkling_heart:

**Pull request merged**